monika.example.yml

```diff
@@ -4,7 +4,7 @@
 probes:
   - id: '1'
     requests:
-      - url: http://example.com
+      - url: http://github.com
 # Configuration example for sending HTML Form Submission
 # Using this configuration Monika will send a POST request to http://www.foo.com/login.php
 # with the defined headers and request's body.
```

Comment:
could you please open other examples also with valid to run configuration.
e.g  - id:  2
          name: HTML form submission
          ...

Reply:
actually the example has a lot of entries commented out, which can be used for examples or templates.